base-template.sh

```diff
@@ -540,9 +639,32 @@ update_shared_hooks_if_appropriate() {
                     echo "$PULL_OUTPUT" >&2
                 fi
             else
-                echo "* Retrieving shared hooks from: $SHARED_REPO"
-                [ -d "$SHARED_ROOT" ] && rm -rf "$SHARED_ROOT"
-                CLONE_OUTPUT=$(git -c core.hooksPath=/dev/null clone "$SHARED_REPO" "$SHARED_ROOT" 2>&1)
+                echo "* Retrieving shared hooks from: $SHARED_REPO_CLONE_URL"
```

Comment:
Do we actually have `$SHARED_REPO_CLONE_URL` set here?

Reply:
Yes `set_shared_root`